elastic/kibana
# [Infrastructure UI] Lens charts reload problem fix

## Summary

closes #149514

This PR fixes the problem with lens charts not always reloading when a search is submitted.

### How to test

Go to Infrastructure > Hosts
- Select a relative date range of 15m from now and click on submit - should always make all lens charts reload
- Select any absolute date and click on submit- should always make all lens charts reload
